jschon/vocabulary/__init__.py

```diff
@@ -51,6 +51,22 @@ def _bootstrap(self, value: Mapping[str, JSONCompatible]) -> None:
             for vocabulary in self.default_vocabularies:
                 self.kwclasses.update(vocabulary.kwclasses)
 
+    def get_kwclass(self, key):
+        try:
+            return self.kwclasses[key]
+        except KeyError:
+            capitalized_name = (
+                key[0].upper() +
+                (key[1:] if len(key) > 1 else "")
+            )
+            unknown_class = type(
+                f'UnknownKeyword{capitalized_name}',
+                (AnnotationKeyword,),
+                {'key': key},
```

Comment:
Nice one! I like this approach.

I know this is really pedantic, but I'd prefer `dict(key=key)` here: I think it makes it a bit more obvious that this becomes the `__dict__` of the new class, and that it is setting a class attribute.

Reply:
Makes sense to me - it's your project, please be as pedantic as you like!